include/FileBrowser.h

```diff
@@ -73,7 +73,9 @@ class FileBrowser : public SideBarWidget
 			const QString & title, const QPixmap & pm,
 			QWidget * parent, bool dirs_as_items = false,
 			const QString& userDir = "",
-			const QString& factoryDir = "");
+			const QString& factoryDir = "",
+			const QStringList& items = {}
```

Comment:
Why store a separate list for the favorite items when you can just fetch it from the config?